mnelab.py

```diff
@@ -428,7 +431,10 @@ def plot_psd(self):
 
     def plot_montage(self):
         montage = mne.channels.read_montage(self.all.current.montage)
-        montage.plot(kind="topomap")
+        if int(mne.__version__.split('.')[1]) >= 15:
```

Comment:
I'd prefer a a try/except block that catches the TypeError when you specify the new keyword argument.